pwa/app/routes/event.$eventKey.tsx

```diff
@@ -370,7 +389,20 @@ function TeamsTab({ teams }: { teams: Team[] }) {
                 {t.city}, {t.state_prov}, {t.country}
               </CredenzaDescription>
             </CredenzaHeader>
-            <CredenzaBody>Full name: {t.name}</CredenzaBody>
+            <CredenzaBody>
+              <h1 className="text-xl">Title</h1>
+              <ScrollArea className="h-[70vh] md:h-auto">
```

Comment:
The scroll doesn't seem to work on the desktop layout for me?

Reply:
In theory it should be able to show you the full match schedule on desktop, perhaps that is a poor assumption. e.g.:

![image](https://github.com/user-attachments/assets/e75b2f63-f28c-4f97-a644-6551cd2612f4)

If I increase my zoom levels though, I can't scroll though:

![image](https://github.com/user-attachments/assets/9ab2f81d-aa8a-491b-b56f-d67b4b491d39)

I'll look at it

Reply:
Fixed